.github/workflows/tests.yaml

```diff
@@ -196,21 +196,6 @@ jobs:
                         build \
                         --copy-artifacts-to objdir-clone \
                      "
-            - name: Run Tests
-              run: |
-                  ./scripts/run_in_build_env.sh \
-                  "./scripts/tests/run_test_suite.py \
-                     --chip-tool ./out/linux-x64-chip-tool${CHIP_TOOL_VARIANT}-${BUILD_VARIANT}/chip-tool \
-                     run \
-                     --iterations 1 \
-                     --test-timeout-seconds 120 \
-                     --all-clusters-app ./out/linux-x64-all-clusters-${BUILD_VARIANT}/chip-all-clusters-app \
-                     --lock-app ./out/linux-x64-lock-${BUILD_VARIANT}/chip-lock-app \
-                     --ota-provider-app ./out/linux-x64-ota-provider-${BUILD_VARIANT}/chip-ota-provider-app \
-                     --ota-requestor-app ./out/linux-x64-ota-requestor-${BUILD_VARIANT}/chip-ota-requestor-app \
-                     --tv-app ./out/linux-x64-tv-app-${BUILD_VARIANT}/chip-tv-app \
-                     --bridge-app ./out/linux-x64-bridge-${BUILD_VARIANT}/chip-bridge-app \
-                  "
 
             - name: Run Tests using the python parser sending commands to chip-tool
               #   https://github.com/project-chip/connectedhomeip/issues/27673
```

Comment:
The loss of "Run Tests" above, is there additional coverage that would be enabled by removing the `if: matrix.build_variant != 'no-ble-tsan-clang'` ?

Reply:
Ideally we would need to enable the `tsar` variant back I agree. IIRC the reason why it has been disabled is not specifically because of a `tsan` issue in itself but for some timeouts. As of today we have not found yet why do we timeout, if this is because if something running slower with `tsan` that creates a timeout or some sort of race because of the aforementioned slowlyness.

I agree that this is not great and I won't argue against blocking this PR until we fix the root cause for making sure we do not introduce new `tsan` issues in some of our backends.